src/features/retreat/components/pirate/RetreatPirate.tsx

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+import { Action } from "components/ui/Action";
+import { GRID_WIDTH_PX, PIXEL_SCALE } from "features/game/lib/constants";
+import { blacksmithAudio } from "lib/utils/sfx";
+import { MapPlacement } from "features/game/expansion/components/MapPlacement";
+import { Modal } from "react-bootstrap";
+import { CloseButtonPanel } from "features/game/components/CloseablePanel";
+import { Inventory } from "features/game/types/game";
+import { GoblinPirateItems } from "./components/GoblinPirateItems";
+import { SUNNYSIDE } from "assets/sunnyside";
+import { NPC } from "features/island/bumpkin/components/DynamicMiniNFT";
+import { Equipped } from "features/game/types/bumpkin";
+
+const bumpkin: Equipped = {
+  body: "Goblin Potion",
+  hair: "White Long Hair",
+  hat: "Pirate Hat",
+  shirt: "Fancy Top",
+  pants: "Pirate Pants",
+  tool: "Pirate Scimitar",
+  background: "Seashore Background",
+  shoes: "Black Farmer Boots",
+};
```

Comment:
I think this NPC is different to the modal, might be worth spreading `...bumpkin` on `CloseButtonPanel`